src/components/profile/MyMissions.js

```diff
@@ -1,5 +1,24 @@
-const MyMissions = () => (
-  <h2>My Missions</h2>
-);
+import Table from 'react-bootstrap/Table';
+import { shallowEqual, useSelector } from 'react-redux';
+
+const MyMissions = () => {
+  const myMissions = useSelector((state) => state.missions, shallowEqual)
+    .filter(({ status }) => status === 'Active member');
```

Comment:
**[optional]**: It will be better to filter your missions items with the map method

```JS
rockets.filter(({ status }) => status === 'Active member').map(({ id, name })=> {})
```